src/templates/app/_layout/header.jsx

```diff
@@ -74,7 +74,8 @@ const Header = () => (
                                         <div />
                                     </div>
                                     <div className='gr-9 gr-hide-m gr-hide-p binary-logo-text'>
-                                        <div />
+                                        <div data-show='-maltainvest' />
+                                        <span data-show='maltainvest' className='binary-type-logo-old'/>
```

Comment:
By changing `span` to `div`:
- It receives same styles, then only `background-image` would be enough, no need to duplicate others.
- Won't need the `display: inline-block;` (this caused 3px increase on height of Header)